### PR TITLE
Made navbar

### PR DESCRIPTION
**Pull request by Imafidizzy**

### CODE REVIEW

File: frontend/src/Browse.js [hunk 1]
```diff
@@ -98,7 +99,7 @@ function Browse() {
           }
         </tbody>
       </table>
-    </div>
+    </div> </>
```
Comment (echang49) [inline frontend/src/Browse.js:102]: What are the empty brackets for

File: frontend/src/Browse.js [hunk 2]
```diff
@@ -59,6 +59,7 @@ function Browse() {
   }
 
   return (
+    <> <Nav />
```
Comment (echang49) [inline frontend/src/Browse.js:62]: What are the empty brackets for

File: frontend/src/ConnectWallet.js [hunk 3]
```diff
@@ -39,7 +39,7 @@ function ConnectWallet() {
           });
   }
     return (
-      <div>
+      <> <Nav /><div>
```
Comment (echang49) [inline frontend/src/ConnectWallet.js:42]: What are the empty brackets for

File: frontend/src/ConnectWallet.js [hunk 4]
```diff
@@ -57,7 +57,7 @@ function ConnectWallet() {
       <div className='footer'>
         this is the footer
         </div>
-      </div>
+      </div></>
```
Comment (echang49) [inline frontend/src/ConnectWallet.js:60]: What are the empty brackets for

File: frontend/src/CreateNFT.js [hunk 5]
```diff
@@ -91,6 +92,8 @@ function CreateNFTListing() {
     }
 
     return (
+        <>
```
Comment (echang49) [inline frontend/src/CreateNFT.js:95]: What are the empty brackets for

File: frontend/src/CreateNFT.js [hunk 6]
```diff
@@ -125,6 +128,7 @@ function CreateNFTListing() {
                 </div>
             </div>
         </div>
+        </>
```
Comment (echang49) [inline frontend/src/CreateNFT.js:131]: What are the empty brackets for

File: frontend/src/Nav.js [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+    useEffect(() => {
+        window.ethereum.request({ method: 'eth_accounts' })
+            .then((accounts) => {
+                if(accounts.length > 0) {
+                    setAuthenticated(true);
+                }
+            })
+    }, []);
+
+  if (authenticated){
```
Comment (echang49) [inline frontend/src/Nav.js:17]: Is there a navbar that theyre supposed to see if theyre not authenticated?

**Review by echang49 — CHANGES_REQUESTED** [review]

File: frontend/src/Browse.js (hunk 1 above)
Comment (Imafidizzy) [reply frontend/src/Browse.js:102]: It did allow me to just add <Nav /> on its own. Either I put it everything within empty brackets. I chose empty brackets Nav and other page components dont ever jumbled due to being in the same div

File: frontend/src/Nav.js (hunk 7 above)
Comment (Imafidizzy) [reply frontend/src/Nav.js:17]: No. Didnt think we needed one

Comment (echang49) [reply frontend/src/Nav.js:17]: We do need one.

**Pull request merged**